common.mk

```diff
@@ -70,6 +70,12 @@ KERNEL_PATH := kernel/sony/msm-$(SOMC_KERNEL_VERSION)
 # Sanitized prebuilt kernel headers
 -include $(KERNEL_PATH)/common-headers/KernelHeaders.mk
 
+# Configure qti-headers auxiliary module via soong so that the correct headers
+# under kernel/sony/msm-X.Y/kernel-headers are chosen
+SOONG_CONFIG_NAMESPACES += qti_kernel_headers
```

Comment:
We got a nice `+=` here to not accidentally mess with what is set in `CommonConfig.mk` for `gralloc`... Fortunately that one uses `+=` too.

Which, by the way, why wasn't that moved from `CamelCase.mk` to product-inheritance `lowercase.mk`?

Reply:
We can probably remove the gralloc thing entirely now, no?
I don't know of any A11 devices that set `TARGET_USES_GRALLOC1`.

Reply:
```console
me:sodp/device/sony $ rg GRALLOC1
yoshino/platform.mk
74:TARGET_USES_GRALLOC1 := true

nile/platform.mk
74:TARGET_USES_GRALLOC1 := true

ganges/platform.mk
71:TARGET_USES_GRALLOC1 := true

loire/PlatformConfig.mk
72:TARGET_USES_GRALLOC1 := true
```

(Note there is no Tone checkout in my tree, neither is Loire relevant)

Reply:
Whoops, had my trees checked on the wrong branches locally. Do you wanna submit the change to lowercase?